assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Overview.js

```diff
@@ -218,101 +250,92 @@ const Overview = ( {
 					</Cell>
 				) }
 
-				{ analyticsModuleActiveAndConnected &&
-					! hasAnalyticsData &&
-					! error && (
-						<Cell { ...halfCellProps }>
-							<WidgetReportZero moduleSlug="analytics" />
-						</Cell>
-					) }
-
-				{ analyticsModuleActiveAndConnected &&
-					hasAnalyticsData &&
-					! error && (
-						<Fragment>
-							<Cell { ...quarterCellProps }>
-								<DataBlock
-									stat={ 2 }
-									className="googlesitekit-data-block--visitors googlesitekit-data-block--button-3"
-									title={ __(
-										'Unique Visitors from Search',
-										'google-site-kit'
-									) }
-									datapoint={ analyticsVisitorsDatapoint }
-									change={ analyticsVisitorsChange }
-									changeDataUnit="%"
-									context="button"
-									selected={ selectedStats === 2 }
-									handleStatSelection={ handleStatsSelection }
-								/>
-							</Cell>
+				{ isAnalyticsGatheringData && ! hasAnalyticsData && ! error && (
+					<Cell { ...halfCellProps }>
+						<WidgetReportZero moduleSlug="analytics" />
+					</Cell>
+				) }
 
-							<Cell { ...quarterCellProps }>
-								{ viewContext === VIEW_CONTEXT_DASHBOARD &&
-									! analyticsGoalsData?.items?.length && (
-										<CTA
-											title={ __(
-												'Use goals to measure success',
-												'google-site-kit'
-											) }
-											description={ __(
-												'Goals measure how well your site or app fulfills your target objectives',
-												'google-site-kit'
-											) }
-											ctaLink={ supportURL }
-											ctaLabel={ __(
-												'Create a new goal',
-												'google-site-kit'
-											) }
-											ctaLinkExternal
-										/>
-									) }
-								{ viewContext === VIEW_CONTEXT_DASHBOARD &&
-									analyticsGoalsData?.items?.length > 0 && (
-										<DataBlock
-											stat={ 3 }
-											className="googlesitekit-data-block--goals googlesitekit-data-block--button-4"
-											title={ __(
-												'Goals',
-												'google-site-kit'
-											) }
-											datapoint={
-												analyticsGoalsDatapoint
-											}
-											change={ analyticsGoalsChange }
-											changeDataUnit="%"
-											context="button"
-											selected={ selectedStats === 3 }
-											handleStatSelection={
-												handleStatsSelection
-											}
-										/>
-									) }
+				{ ! isAnalyticsGatheringData && hasAnalyticsData && ! error && (
```

Comment:
```suggestion
				{ ! isAnalyticsGatheringData && ! error && (
```

Reply:
What about?
```suggestion
				{ hasAnalyticsData && ! error && (
```

Since we are checking `isSearchConsoleGatheringData === false` in `hasAnalyticsData`.

Reply:
Right, but we still want to render this expression if there is no data as mentioned before. Zero data is okay here but it shouldn't be gathering.